### PR TITLE
ci(tests): historacle e2e

### PR DESCRIPTION
**Pull request by adamewozniak**
## Description

Historacle e2e tests

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by adamewozniak — COMMENTED** [review]
looks good to me!

**Review by zarazan — APPROVED** [review]
Great code 👍🏻

**Comment by adamewozniak** [comment]
Lint issues are happening here just because: https://github.com/golangci/golangci-lint-action/issues/244

`make lint` works locally all good

File: tests/e2e/e2e_setup_test.go [hunk 1]
```diff
@@ -242,12 +244,14 @@ func (s *IntegrationTestSuite) initGenesis() {
 	var bech32GenState bech32ibctypes.GenesisState
 	s.Require().NoError(cdc.UnmarshalJSON(appGenState[bech32ibctypes.ModuleName], &bech32GenState))
 
+	// bech32
```
Comment (robert-zaremba) [inline tests/e2e/e2e_setup_test.go:247]: why adding these comments? it's "purple prose"

**Pull request merged**